flake.lock

```diff
@@ -0,0 +1,77 @@
+{
+  "nodes": {
+    "flake-compat": {
+      "flake": false,
+      "locked": {
+        "lastModified": 1641205782,
+        "narHash": "sha256-4jY7RCWUoZ9cKD8co0/4tFARpWB+57+r1bLLvXNJliY=",
+        "owner": "edolstra",
```

Comment:
I'm thinking that we should probably pin our own nixpkgs fork in this PR, just to make things easier on people reading up on it, and also freeze the changes for this particular repo.
WDYT?

Reply:
I do not see any practical difference, as long as we are reaching github for the code.
The flake.lock mechanism makes sure that we are hardly pinned on specific snapshot of nixpkgs world, regardless of its journey forward. 

However I can easily see, how in the future, only our Github Org repo are whitelisted and we need to adjust to that reality. Until that point, I would just keep things simple.